packages/core/src/Stores/client-store.js

```diff
@@ -67,6 +67,7 @@ export default class ClientStore extends BaseStore {
         decimal_places: undefined,
     };
 
+    @observable connected_apps = [];
```

Comment:
Could this state be kept "locally" in the <ConnectedApps /> component instead of globally in the client store? Or is there a specific reason as to why we need this in the client-store?

Reply:
I think it is a part of our client data so, I rather kept it here.
what do you think? should we move it to the local file?

Reply:
@arash-fs Yes I would say so since this state is not used in other places "outside of" connected apps (and client-store is big enough already).

This article explains it better than me: https://kentcdodds.com/blog/state-colocation-will-make-your-react-app-faster